web/packages/design/src/Menu/Menu.jsx

```diff
@@ -31,20 +30,12 @@ const POSITION = {
 };
 
 class Menu extends React.Component {
-  getContentAnchorEl = () => {
-    if (this.menuListRef.selectedItemRef) {
-      return ReactDOM.findDOMNode(this.menuListRef.selectedItemRef);
```

Comment:
I don't know what `selectedItemRef` was. Probably a leftover from earlier refactors? There's no history for this fragment. `selectedItemRef` is not something available in the DOM, it must've been some custom thing set by our React code.